pkg/storegateway/indexheader/reader_pool_test.go

```diff
@@ -36,7 +38,7 @@ func TestReaderPool_NewBinaryReader(t *testing.T) {
 		},
 		"lazy reader and close on idle timeout are both enabled": {
 			lazyReaderEnabled:     true,
-			lazyReaderIdleTimeout: time.Minute,
+			lazyReaderIdleTimeout: 2 * time.Minute,
```

Comment:
What's the reason for this change?

Reply:
Accidentally committed the local test. Will revert.